### ISSUE TITLE
Certain "claims" field disappear after a period of time.

### ISSUE THREAD

**Issue by ChuckJonas** [okta/okta-react#168]
## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior

I'm using information from the idToken.claims in the react application:

``` 
const { authState } = useOktaAuth();
console.log(authState?.idToken?.claims);
```

When I first log in, the claims hold the following information:

```
at_hash: "REPLACED"
aud: "REPLACED"
auth_time: 1633533583
email: "test@example.com"
exp: 1633544090
iat: 1633540490
idp: "REPLACED"
iss: "https://dev-96453639.okta.com/oauth2/default"
jti: "ID.REPLACED"
name: "John Doe"
nonce: "REPLACED"
preferred_username: "test@example.com"
sub: "REPLACED"

```

After a short period of time, some of the claims fields seem to disappear:

```
amr: ["pwd"]
at_hash: "REPLACED"
aud: "REPLACED"
auth_time: 1633533583
email: "test@example.com"
exp: 1633544090
iat: 1633540490
idp: "REPLACED"
iss: "https://dev-96453639.okta.com/oauth2/default"
jti: "REPLACED"
nonce: "REPLACED"
sub: "REPLACED"

```
(note: `name`, `preferred_username`, etc no longer show up)

The access_token is not expired (still passes verification on our API).

## Expected behavior

I would expect these would remain the same as long as the authentication has not expired.

## Environment

- Package version:  6.1.0
- React version: 17.0.2
- Browser: Brave (shields turned off)
- OS: OSX
- Node version (`node -v`): v12.22.2
- Other:

## Configuration

- Okta application type: Single Page App (SPA)

**Comment by aarongranick-okta** [comment]
@ChuckJonas This issue should be fixed in `okta-auth-js` version `5.7.0` or higher.

**Comment by aarongranick-okta** [comment]
Closing this issue as fixed. Feel free to re-open or create a new issue should the problem appear in current version of `okta-auth-js`
